packages/avatar/src/Avatar.js

```diff
@@ -165,14 +222,31 @@ class Avatar extends Component {
 
   render() {
     const { size, name, ...otherProps } = this.props;
+    const { firstName, lastName } = otherProps;
     const { className } = otherProps;
     const { imageUrl, hasImageError } = this.state;
     const { handleImageError } = this;
-    const backgroundId =
-      backgroundIdFromName(name) <= COLOR_VARIANT_COUNT
-        ? backgroundIdFromName(name)
-        : COLOR_VARIANT_COUNT;
-    const label = `Avatar for ${name}`;
+    let nameObj = {};
```

Comment:
Rather than declaring an empty object with `let` and then the `if/else` statement below, I think you could instead use `const` and then a ternary to define it based on your `(!firstName && !lastName)` condition.

Reply:
Good point, updated.